imagebuildah/stage_executor.go

```diff
@@ -1168,6 +1193,22 @@ func (s *StageExecutor) Execute(ctx context.Context, base string) (imgID string,
 			if err != nil {
 				return "", nil, fmt.Errorf("error checking if cached image exists from a previous build: %w", err)
 			}
+			// All the best effort to find cache on localstorage have failed try pulling
+			// cache from remote repo if `--cache-from` was configured.
+			if cacheID == "" && s.executor.cacheFrom != "" && cacheKey != "" {
+				src := fmt.Sprintf("%s:%s", s.executor.cacheFrom, cacheKey)
+				id, err := s.pullCache(ctx, src)
```

Comment:
`id` and `err` don't get used outside of this block, so merging the `pullCache` call into the `if` statement just below will shorten it up.